controllers/articles.controller.js

```diff
@@ -59,3 +60,24 @@ exports.addComment = (req, res, next) => {
       next(err);
     });
 };
+
+exports.patchArticle = (req, res, next) => {
+  const { article_id } = req.params;
+  const { inc_votes } = req.body;
+
+  const updateArticleQuery = updateArticle({ article_id, inc_votes });
+  const queries = [updateArticleQuery];
+
+  if (article_id) {
```

Comment:
psql gives an error if the article id is not a number - if it is a number but does exist it will return nothing in the model so we can get the error this way